### ISSUE TITLE
Umbraco Heartcore | Feature request | Add form ID to GraphQL API response

### ISSUE THREAD

**Issue by anthwinter** [umbraco/Umbraco.Cloud.Issues#439]
### Issue description

We are currently using the GraphQL API to query content with Umbraco Heartcore, but have an issue while also using Forms.

We have a content page that has a field on it that references one of our "forms", but we are unable to retrieve the form via the GraphQL API as it just returns `null`

![Screenshot 2021-09-10 at 17 12 07](https://user-images.githubusercontent.com/4440999/132884678-c7235753-72fe-454b-95a3-ad6f7481f952.png)

This means we are unable to know which form is attached to the page.

It would be great if we could at least get the form `id` returned on this property, so that we can then go straight to `GET`ing it via the REST API where we are able to query forms.

**Comment by sitereactor** [comment]
@Frost117 are you able to help reproduce this issue? The form should be included as part of the data returned through this query, but if we can reproduce the issue its obviously something we need to fix.

**Comment by RyuLindow** [comment]
I can reproduce it on a different project - [VIDEO](https://www.dropbox.com/s/yw3w6nw460iqu3j/fPQujydt7O.mp4?dl=0)
![image](https://user-images.githubusercontent.com/56826332/134008214-11eb3cf5-84a4-4df7-8c8b-deca9cf44a02.png)

**Comment by rasmusjp** [comment]
Hi @anthwinter 

We've deployed a fix so the form is now returned

**Comment by anthwinter** [comment]
Hi @rasmusjp. It looks like some of our pages have this working now, but others don't and still return `null` (within the same project).

I've checked to see if it was just that I needed to re-add the form in the "Content" area and re-publish it, but this didn't help either.

An example would be that we have a Generic Page and a Email Form Page. They both have forms set up on them the same way, using Nested Content and Form Picker types, but Email Form Page returns `null` for the form JSON
